docs/elements.md

```diff
@@ -704,22 +704,22 @@ generation if two (or more) choices are identical.
-| `external-json`               | string  | special | Optional path to a JSON file to load external answer choices from. Answer choices are stored as lists under "correct" and "incorrect" key names.                               |
-| `external-json-correct-key`   | string  | special | Optionally override default json "correct" attribute name when using `external-json` file.                                                                                     |
-| `external-json-incorrect-key` | string  | special | Optionally override default json "incorrect" attribute name when using `external-json` file.                                                                                   |
-| `allow-blank`                 | boolean | false   | Whether or not an empty submission is allowed. If `allow-blank` is set to `true`, a submission that does not select any option will be marked as incorrect instead of invalid. |
+| Attribute                     | Type                                      | Default  | Description                                                                                                                                                                    |
+| ----------------------------- | ----------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `answers-name`                | string                                    | —        | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question.           |
+| `weight`                      | integer                                   | 1        | Weight to use when computing a weighted average score over elements.                                                                                                           |
+| `display`                     | "block", "inline", or "dropdown"          | "inline" | How to display the input field.                                                                                                                                                |
+| `number-answers`              | integer                                   | special  | The total number of answer choices to display. Defaults to displaying one correct answer and all incorrect answers.                                                            |
+| `sort`                        | "random", "ascend", "descend", or "fixed" | "random" | Order to display answer choices.                                                                                                                                               |
```

Comment:
Ditto. `sort="fixed"` may not be trivially understood.

Reply:
Do you think the name `sort` for this attribute is clear? I thought about changing it but wasn't sure if anyone had any strong opinions.

Reply:
Good point. `pl-order-blocks` uses `source-blocks-order`. Maybe use `order`?

Reply:
I agree, order is a little clearer than sort.

Reply:
Name changed here: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/43bdad261b930462316f8fd5e00b162f2b040978